src/Heading.js

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+function Heading() {
```

Comment:
Nice work!

Reply:
Thank Ali for following up. 
I have uploaded the requirement of lesson 2 now, It will be appreciated if you can check.

Regards,